packages/now-cli/test/integration.js

```diff
@@ -120,6 +120,11 @@ if (!process.env.CI) {
   globalDir = path.join(tmpDir.name, 'com.vercel.tests');
 
   defaultArgs.push('-Q', globalDir);
+
+  if (process.env.VERCEL_SCOPE) {
+    defaultArgs.push('--scope', process.env.VERCEL_SCOPE);
+  }
```

Comment:
Hmm, I'm not sure this is a good idea since we have tests for `--scope` already.

See the test `try to deploy with non-existing team` for example. We don't want to pass two `--scope` options.